astroplan/constraints.py

```diff
@@ -146,12 +190,14 @@ def __init__(self, min=None, max=None):
             self.max = max
 
```

Comment:
I commented on some other file about this, but meant to comment within the PR here. 

The bug that's failing the ci tests comes from the test that tries to access an attribute on `AltitudeConstraint` that was never set. [Here](https://github.com/astropy/astroplan/blob/0303f571c204e0a867538fcec4f14ceb636c8049/astroplan/constraints.py#L195) you call `self.boolean_constraint`, but the `__init__` method doesn't have `self.boolean_constraint = boolean_constraint` in it (like `AirmassConstraint` does, [for example](https://github.com/astropy/astroplan/blob/0303f571c204e0a867538fcec4f14ceb636c8049/astroplan/constraints.py#L234)).